antenna_offset: [200,2, 0.0,-0.45,0.0,  0.0,0.45,0.0,
                 201,2, -0.6,0.45,0.0, -0.6,-0.45,0.0]
ranging_bias: [200, -0.75,  201, -0.75]
locked_anchor_ids: []

cauchy_scale: 0.5
huber_delta: 0.5

distance_error: 0.7
write_csv: false     